politeiawww/backend.go

```diff
@@ -90,6 +90,9 @@ type backend struct {
 
 	// Count of user proposals
 	numOfPropsByUserID map[string]int
+
+	// User vote action on each comment
+	userVoteActionByCommentID map[string]map[string]int64 // [token+userid][commentid]action
```

Comment:
We flip back and forth between referring to comment upvotes/downvotes as `votes` and `likes`.  For example, d has a `decredPluginCommentsLikesCache` and a www inventory record has a `commentLikes` field, but we refer them as `votes` right here.

Do you think it would make it easier to read if we were consistent and referred to them as `likes` everywhere instead of `votes`?